Rocket.Chat/Managers/UserReviewManager.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  UserReviewManager.swift
+//  Rocket.Chat
+//
+//  Created by Augusto Falcão on 9/14/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+import StoreKit
+
+class UserReviewManager {
+    static let shared = UserReviewManager()
+
+    let nextDateForReviewKey: String = "kNextDateForReview"
+
+    private let defaults = UserDefaults.standard
+
+    var nextDateForReview: Date {
+        get {
+            if let date = defaults.object(forKey: nextDateForReviewKey) as? Date {
+                return date
+            } else {
+                let date = calculateNextDateForReview()
+                defaults.set(date, forKey: nextDateForReviewKey)
+                return date
+            }
+        }
+        set(newDate) {
+            defaults.set(newDate, forKey: nextDateForReviewKey)
+        }
+    }
+
+    var availableForReview: Bool {
+        return nextDateForReview < Date()
+    }
+
+    func calculateNextDateForReview() -> Date {
+        let week: TimeInterval = 30 // 604800
```

Comment:
You meant to be `30` or you want `604800`?

Reply:
Also, let's add this as a constant? This way we can use the same value in our tests :-)

Reply:
The correct value is 604800, this is the time of a week in seconds. I used 30 seconds to test with simulator.

It is ok the week value be a constant.